docs/wiki/00 - SERVER INSTALL AND CONFIGURATION/01 - LORIS Install/Ubuntu/README.md

```diff
@@ -1 +1,182 @@
+```
+
+This will create a folder called something like `aces-Loris-9e30cf0`. (The
+part after `aces-Loris` is not important).
+
+Rename this file to `loris`, e.g.
+
+```bash
+mv aces-Loris-9e30cf0 loris
+```
+
+Then move this folder to the web root (e.g. `/var/www/`) and go to this 
+directory.
+
+### If installing LORIS for production
+
+```bash
+cd `/var/www/loris/`; make
+```
+
+### If installing LORIS for development
+
+```bash
+# This is necessary for 'phan', a static analysis tool
+sudo pecl install ast-1.0.3
+cd `/var/www/loris/`; make dev
+```
+
+The `dev` target for `make` will install additional libraries that are needed 
+for development process but not for production installs.
+
+## Running the install script
+
+The next step in setting up LORIS is running the script `install.sh` in the 
+`tools/` directory. The script must be run from this directory.
+
+This will begin an interactive setup process that will configure files and
+permissions required to get LORIS up and running.
+
+The install script will prompt for a `root` password in order to configure
+certain file permissions as well as Apache. However, the script should not
+be run as `root` and will exit automatically if it detects that it has been
+executed with superuser permissions.
+
+```bash
+cd tools/
+./install.sh
+```
+
+## Configuring the database
+
+The install script will tell you to navigate to <loris-url>/installdb.php.
+
+This web page will prompt you for the following information:
+
+ * `Server Hostname` localhost if your database is hosted on your VM or the IP address of your database server
+
+ * `Admin Username` e.g. `root` or a database user with permission to create databases and tables
+
+ * `Admin Password` The password for the above database user.
+
+ * `Database Name` set to "LORIS" by default, can be customised
+
+Click submit, and on the next screen that is presented, follow instructions to enter the username and password of your LORIS database user and front-end `admin` user.
+
+If you encounter issues creating/generating your config file, you may have to manually paste the xml output that appears on the screen into the file `/var/www/loris/project/config.xml`
+
+Your LORIS instance should now be accessible by pointing your browser URL to `http://%IPADDRESS%`
+
+If there are any errors or you get a blank page, troubleshoot the errors in your apache error log - by default
+ `/var/log/apache2/loris-error.log`
+
+## Configuring MySQL
+
+Please [go here](https://github.com/aces/Loris/wiki/Hosting-the-Database-Myself)
+for information on configuring MySQL.
+
+### Development
+
+The name of our database filled with dummy test data is "Raisinbread". This 
+can be used by developers to populate LORIS with useable data for testing
+changes to the codebase.
+
+Instructions on installing Raisinbread can be found in the LORIS root folder
+at `raisinbread/README.md`
+
+## Troubleshooting
+
+If errors are encountered during the section "Running the install script",
```

Comment:
I'd like to see here quick links to top 3 troubleshooting questions we get right after Install, which aren't script errors. i.e. What to do if : 
* front end doesn't load -> check/fix config paths via the mysql back-end
* css looks busted (if that is still happening) -> check/fix config paths via the mysql back-end
* you need to reset your admin password -> run the tools script

Reply:
I think we should abstract the troubleshooting document into its own file. These errors aren't specific to Ubuntu so they shouldn't be in the Ubuntu instructions.

Reply:
looks like this is #5920 - new issue.